hooks.go

```diff
@@ -48,11 +48,27 @@ func randomString(len int) string {
 	return string(bytes)
 }
 
-// HookHandler contains the hook, the SCM and the HTTP response handler.
+func storePayload(payloadContent string, hh *HookHandler) (string, error) {
```

Comment:
a little comment please ? (starting with the function name, so the godoc parser is happy)

Reply:
`storePayload` stores the payload.